### DISCUSSION TITLE
Add support for Postfix in the Email transport

### DISCUSSION THREAD

**Discussion by Pachat** [directus/directus#16492] [Feature Requests]
Are there any reasons why Postfix  is not a valid `EMAIL_TRANSPORT` for Directus?

**Comment by rijkvanzanten** [comment]
Heya! Mostly because.. it wasn't implemented (yet) 😄 I've converted this to a feature request 👍🏻 If it gets enough traction it might make it's way into core!

**Comment by paescuj** [comment]
> Are there any reasons why Postfix is not a valid `EMAIL_TRANSPORT` for Directus?

How do you plan to use Postfix with Directus? Actually, Postfix provides the `sendmail` binary, so you should be able to go with `EMAIL_TRANSPORT=sendmail`.

**Reply by Pachat** [reply]
Yes indeed. `EMAIL_TRANSPORT=sendmail`. works with Postfix installed (and no Sendmail installed per se)
But Directus complains with `EMAIL_TRANSPORT=postfix`, hence my question.

**Reply by paescuj** [reply]
Okay, why do you need `EMAIL_TRANSPORT=postfix` then? Do you expect it to do something different?

**Reply by Pachat** [reply]
In fact, no. I was just surprised not to find `postfix `among the allowed `EMAIL_TRANSPORT`s

**Reply by paescuj** [reply]
All right 😃👍

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
